Dockerfile

```diff
@@ -50,23 +67,32 @@ RUN \
  && pip3 install kafka-python==1.3.2 \
  && pip3 install bokeh==0.12.4
 
+# Create a directory to
+RUN mkdir ~/.ssh 
+COPY ./config/ssh_config .ssh/config
```

Comment:
git reads the keys from the .ssh folder from your home folder . Hence we need to have the ssh keys in that folder in order to execute the next command . I am using this branch for D3 and flask development .